pkgs/by-name/st/stown/package.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  lib,
+  pkgs,
```

```suggestion
  tree,
```
Please call attributes directly in Nixpkgs. The `callPackage` function used internally will automatically find the correct attribute in the package set.